### PR TITLE
resolve JIRA-1495

### PR DESCRIPTION
**Pull request by nonstop-qfchen**
Please kindly review the fix to JIRA1495. 

Thanks!

### CODE REVIEW

File: core/sql/regress/tools/runallsb [hunk 1]
```diff
@@ -42,6 +42,7 @@ export SQLMX_REGRESS=1
 export SEABASE_REGRESS=2
 
 TEST_SUBDIRS="core compGeneral executor seabase fullstack2 charsets qat catman1 udr hive" 
+TEST_SUBDIRS="catman1 udr hive"
```
Comment (DaveBirdsall) [inline core/sql/regress/tools/runallsb]: Not sure why this change is here?

File: core/sql/sqlcomp/nadefaults.cpp [hunk 2]
```diff
@@ -3522,8 +3522,10 @@ XDDkwd__(SUBQUERY_UNNESTING,			"ON"),
  XDDui___(USTAT_MAX_READ_AGE_IN_MIN,            "5760"),
   DDui___(USTAT_MAX_SAMPLE_AGE,                 "365"),  // For R2.5 set to a year so user created samples won't be removed.
 
+  DDflt0_(USTAT_MIN_CHAR_DATASIZE_FOR_IS,      "1000"),  // max data size in MB for char type to use 
+                                                         // internal sort without checking UEC.
```
Comment (DaveBirdsall) [inline core/sql/sqlcomp/nadefaults.cpp]: Why does the name of the default talk of "MIN", but the comment talks of "max"?

File: core/sql/regress/tools/runallsb (hunk 1 above)
Comment (nonstop-qfchen) [inline core/sql/regress/tools/runallsb]: Sorry. I'll  restore the original version. Thanks for catching it.

On Mon, Sep 28, 2015 at 11:03 AM, DaveBirdsall <notifications@github.com>
wrote:

> In core/sql/regress/tools/runallsb
> <https://github.com/apache/incubator-trafodion/pull/94#discussion_r40569992>
> :
>
> > @@ -42,6 +42,7 @@ export SQLMX_REGRESS=1
> >  export SEABASE_REGRESS=2
> >
> >  TEST_SUBDIRS="core compGeneral executor seabase fullstack2 charsets qat catman1 udr hive"
> > +TEST_SUBDIRS="catman1 udr hive"
>
> Not sure why this change is here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/94/files#r40569992>.
>



-- 
Regards, --Qifan

**Comment by DaveBirdsall** [comment]
I'm guessing the change to runallsb was not intended?

File: core/sql/regress/tools/runallsb (hunk 1 above)
Comment (nonstop-qfchen) [inline core/sql/regress/tools/runallsb]: Fixed.

On Mon, Sep 28, 2015 at 11:04 AM, Qifan Chen <qifan.chen@esgyn.com> wrote:

> Sorry. I'll  restore the original version. Thanks for catching it.
>
> On Mon, Sep 28, 2015 at 11:03 AM, DaveBirdsall <notifications@github.com>
> wrote:
>
>> In core/sql/regress/tools/runallsb
>> <https://github.com/apache/incubator-trafodion/pull/94#discussion_r40569992>
>> :
>>
>> > @@ -42,6 +42,7 @@ export SQLMX_REGRESS=1
>> >  export SEABASE_REGRESS=2
>> >
>> >  TEST_SUBDIRS="core compGeneral executor seabase fullstack2 charsets qat catman1 udr hive"
>> > +TEST_SUBDIRS="catman1 udr hive"
>>
>> Not sure why this change is here?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/apache/incubator-trafodion/pull/94/files#r40569992>.
>>
>
>
>
> --
> Regards, --Qifan
>
>


-- 
Regards, --Qifan

**Comment by nonstop-qfchen** [comment]
not at all.

On Mon, Sep 28, 2015 at 11:07 AM, DaveBirdsall <notifications@github.com>
wrote:

> I'm guessing the change to runallsb was not intended?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/94#issuecomment-143791095>
> .
>



-- 
Regards, --Qifan

**Comment by nonstop-qfchen** [comment]
Hi Dave,

Thanks a lot of the quick review. I have submitted the rework. Please
review.

Thanks --Qifan

On Mon, Sep 28, 2015 at 11:08 AM, Qifan Chen <qifan.chen@esgyn.com> wrote:

> not at all.
>
> On Mon, Sep 28, 2015 at 11:07 AM, DaveBirdsall <notifications@github.com>
> wrote:
>
>> I'm guessing the change to runallsb was not intended?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/apache/incubator-trafodion/pull/94#issuecomment-143791095>
>> .
>>
>
>
>
> --
> Regards, --Qifan
>
>


-- 
Regards, --Qifan

**Comment by DaveBirdsall** [comment]
The reworked changes look good to me.
